src/pages/Curation/components/PresentRecommendation/index.tsx

```diff
@@ -155,7 +189,7 @@ const PresentRecommendation = () => {
           </h2>
           <Container>
             <Slider {...sliderSettings} className={styles.slider}>
-              {productType.map((product: any) => {
+              {productType?.map((product: any) => {
```

Comment:
any타입과 optional chaining은 제거해도 좋을 것 같습니다.
```suggestion
              {productType.map(product => {
```

Reply:
optional chaining을 걸지 않았을때 productType is undefined 에러가 나서 저렇게 지정해놨습니다~